specify/specify7
# Add a tool to bulk move prepartions from one node to another

Fixes #4042

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Select a storage node that contains 1 or more preparations
- Click "Move Items" (Truck icon)
- Select the new location to move the items to
- Click "Move items from {Current Location} here" in the toolbar

- [ ] Verify items are moved
